examples/process-persistence/README.md

```diff
@@ -0,0 +1,373 @@
+> configuration
+
+### The _"New Hiring Offer"_ Decision (DMN)
+
+This example makes use of the _New Hiring Offer_ DMN to generate a base offer for the `Candidate`. The DMN looks like
+this:
+
+In this simple DMN we have an `Offer` _Decision_, that will generate the candidate offer, which
+has a requirement of a `CandidateData` _Input Data_.
+
+<div style="text-align:center">
+   <figure>
+      <img width=55%  src="docs/images/new_hiring_offer_dmn.png" alt="DMN Diagram">
+      <br/>
+      <figcaption>New Hiring Offer DMN diagram</figcaption>
+   </figure>
+</div>
+
+The DMN defines the following data types (`tCandidateData` & `tOffer` ) matching the Java POJOs defined in the project
+(`CandidateData.java` & `Offer.java`):
+
+<div style="text-align:center">
+   <figure>
+      <img width=49%  src="docs/images/new_hiring_offer_dmn_types_tCandidateData.png" alt="DMN Type Definitions">
+      <img width=49%  src="docs/images/new_hiring_offer_dmn_types_tOffer.png" alt="DMN Type Definitions">
+      <br/>
+      <figcaption>New Hiring Offer DMN types</figcaption>
+   </figure>
+</div>
+
+As expected, `CandidateData` Input and `Offer` Decision have a `tCandidateData` type
+
+The `Offer` Decision uses the following _Boxed Expression_ to generate the `tOffer`:
+
+<div style="text-align:center">
+   <figure>
+      <img width=75%  src="docs/images/new_hiring_offer_dmn_decision.png" alt="DMN Decision">
+      <br/>
+      <figcaption><i>"New Hiring Offer"</i> DMN Decision</figcaption>
+   </figure>
+</div>
+
+### The Java models
+
+The **Hiring** process uses two POJOs to handle the process data, both of them can be found in the _org.kie.kogito.hr_
+package.
+
+The `CandidateData` POJO is the input of the process. It represents the person that wants to get the job.
+
+```java
+public class CandidateData {
+
+    private String name; // Name of the candidate
+    private String lastName; // Last name of the candidate
+    private String email; // Email of the candidate
+    private Integer experience; // Years of experience
+    private List<String> skills; // List of technical skills
+
+    // Constructors, setters, getters...
+}
+```
+
+The `Offer` POJO is the output of the process and represents the job offer that will be sent to the candidate.
+It will be automatically calculated during the process execution depending on the candidate years of experience &
+skills.
+
+```java
+public class Offer {
+
+    private String category; // Job category based on the candidate experience
+    private Integer salary; // Salary based on the candidate experience and skills
+
+    // Constructors, setters, getters...
+}
+```
+
+---
+
+## Running
+
+### Prerequisites
+
+- Java 17+ installed
+- Environment variable JAVA_HOME set accordingly
+- Maven 3.9.6+ installed
+- Docker and Docker Compose to run the required example infrastructure.
+
+And when using native image compilation, you will also need:
```

Comment:
I don't think we need to provide details about native image compilation in this example.